src/worker/packages/server/src/database/database.connection.ts

```diff
@@ -1,13 +1,13 @@
-import { DataSource } from 'typeorm';
+import {DataSource} from 'typeorm';
 
-const AppDataSource = new DataSource({
+const DatabaseSource = new DataSource({
     type: 'postgres',
     host: 'localhost',
-    port: 52313,
+    port: 7432,
     username: 'postgres',
-    password: 'mysecretpassword',
-    database: 'test-typescript-example',
+    password: 'A79Vm5D4p2VQHOp2gd5',
+    database: 'activepieces',
     synchronize: true,
 });
 
-export default AppDataSource;
\ No newline at end of file
+export default DatabaseSource;
```

Comment:
Can we please agree on using named exports instead of default exports? Named exports are better:
- Default exports can be named anything when imported, i.e. `DatabaseSource` may well be imported as `Cat`
- IDEs can suggest or auto-import named imports
- Linters warn when a named import doesn't exist

I think we should add this rule to our conventions doc.

Reply:
Done